### ISSUE TITLE
incompatibility with mods

### ISSUE THREAD

**Issue by AkuseruGH** [apavarino/Deadchest#56]
I have no idea if you guys are interested in this, but on servers with plugins and mods when using curios api and cosmetic armor, deadchest doesn't save items that were placed in the respective mods inventories. 

It's just a bug that I found and I think that for obvious reasons it doesn't work

**Comment by apavarino** [comment]
Yes it's impossible to provide compatibility whith every plugins :(
